src/InterpreterGenerator.h

```diff
@@ -415,6 +416,14 @@ class NodeGenerator : public RamVisitor<std::unique_ptr<InterpreterNode>> {
         return std::make_unique<InterpreterNode>(I_Exit, &exit, std::move(children));
     }
 
+    NodePtr visitCall(const RamCall& call) override {
+        std::vector<size_t> data;
+        auto subs = program->getSubroutines();
+        size_t i = distance(subs.begin(), subs.find(call.getName()));
```

Comment:
As for `data` above, what is `i`? It looks like it might be some sort of distance metric from the method name. Can the name be made clearer (possibly then reflected in the rename of the container `data`.

Reply:
We identify the subroutine by its position in the subroutine collection. I will add more comments.